frontend/app/src/scenes/Home/scenes/Ambassadors/index.tsx

```diff
@@ -16,49 +17,81 @@ export interface Ambassador {
 interface AmbassadorsStateProps {
   ambassadors: Ambassador[];
   isLoading: boolean;
+  pagination: {
+    current: number,
+    pageSize: number,
+    total: number,
+  };
 }
 
+export interface AmbassadorPayload {
+  content: Ambassador[];
+  totalElements: number;
+}
+const colStyles = { fontSize: 16, fontWeight: 2 };
```

Comment:
Move CSS to an external style sheet.

create a css file named styles.css on the same directory.

Reply:
ex: https://github.com/sef-global/sef-core/blob/c64a11e8cd9bf3a131f54f0f79bc028f240a2708/frontend/app/src/scenes/Login/index.tsx#L13

(Btw, you don't need to add `:local` for classes. the webpack has already configured the CSS scope as local)

Reply:
ok